features/issue/components/issuesList/IssuesList.tsx

```diff
@@ -1,21 +1,35 @@
 'use client'
 
-import React, { useState } from 'react'
+import React, { useState, useEffect, useTransition } from 'react'
 import { Accordion } from '@/components/ui/accordion'
 import { IssuesListItem } from './IssuesListItem'
 import { useToast } from '@/components/ui/use-toast'
-import { useGetIssues } from '../../api'
 import { LoadingSpinner } from '@/components/ui/LoadingSpinner'
 import { Searchbar, StatusFilter } from './issuesFilter'
 import { PaginationBar } from '@/shared'
+import { getIssues } from '@/actions/getIssues'
+import { Issue } from '../../types'
 
 export const IssuesList = () => {
```

Comment:
To samo co było napisane w editIssueForm

https://nextjs.org/docs/app/building-your-application/routing/loading-ui-and-streaming#example